autoload/dirvish.vim

```diff
@@ -83,6 +83,7 @@ function! dirvish#shdo(l1, l2, cmd)
   write
   if executable('chmod')
     call system('chmod u+x '.tmpfile)
+    edit
```

Comment:
> I do want to remove most of the usages of noautocmd in dirvish.

:+1: 

> to "touch" a buffer to force it to be the "alternate" buffer

We is that necessary (I can imagine)?
It is not because you are not using `win_gotoid` etc - i.e. you really want to change what is the alternate buffer.  Makes sense to not have autocommands in that case.